### DISCUSSION TITLE
KaJam - 7 day Game Jam with $16,000 in Prizes

### DISCUSSION THREAD

**Discussion by slmjkdbtl** [replit/kaboom#231] [Announcements]
![kajam](https://user-images.githubusercontent.com/9929523/134581551-170523bf-284e-4fa6-8743-115fdee6102f.png)

Replit is hosting a game jam [KaJam](https://replit.com/kajam) with fat prizes. Join!

**Comment by lajbel** [comment]
Happy Jammin to all!!!! 💞🎉
